CMakeLists.txt

```diff
@@ -6,13 +6,14 @@
 # =============================================================================
 
 cmake_minimum_required(VERSION 3.3.0 FATAL_ERROR)
-project(NMODL CXX)
+project(
+  NMODL
+  VERSION 0.2
+  LANGUAGES CXX)
```

Comment:
just a nitpick : does cmake-format force splitting this on multiple line? quite superfluous !

Reply:
Yes, I agree ;)

Reply:
@alkino : note there is `# ~~~` marker in cmake-format to disable it (and avoid absurd formatting)

Reply:
This is the default rule of no more than 3 words in a command on the same line. Maybe we can change it to 10?

Reply:
do you mean number of args settings? I don't know if there is an option but I would say why not!